flash/core/model.py

```diff
@@ -889,3 +890,33 @@ def serve(self, host: str = "127.0.0.1", port: int = 8000, sanity_check: bool =
         composition = Composition(predict=comp, TESTING=flash._IS_TESTING)
         composition.serve(host=host, port=port)
         return composition
+
+    def to_onnx(self, file_path: Union[str, Path], input_sample: Optional[Tensor] = None, **kwargs):
```

Comment:
Could you add some tests ?

Reply:
Yeah sure thing when I get back to work tomorrow. I'll probably just modify the tests already present in the lightning repo.